pydantic_extra_types/mongo_objectId.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any
+
+from bson import ObjectId
+from pydantic_core import core_schema
+from pydantic_core import PydanticCustomError
+
+
+class ObjectIdField(str):
```

Comment:
Is there any issue with naming this class `MongoId` or `MongoObjectId`? Sounds like a better name to me.. WDYT? and I would possibly avoid using the Field suffix unless deemed necessary

Reply:
LGTM @Ale-Cas `MongoObjectId` is a name I prefer over `ObjectIdField` as it is more specific, indicating its usage within the MongoDB environment.

Reply:
@SkandaPrasad-S this change was not applied why was it marked as resolved? 
do you mind if I resolve the threads myself once i reviewed pls? 🙏